docs/_pages/release-process.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Release process"
+permalink: /release-process/
+toc: false
+---
+
+The following steps must be executed to release a new version of Phoenicis:
+* Create release branches (e.g. "5.0-alpha") for phoenicis and scripts
+* Protect release branch
+    * Require pull request reviews before merging
+    * Require status checks to pass before merging (Travis CI and Codacy)
+* on the release branch:
+    * Add release branch to `branches` section of `.travis.yml` such that Travis CI executes checks for the branch
+    * Specify scripts release branch in configuration (`application.repository.default.git.url`)
+    * Set release version for Maven in `pom.xml` files
+    * Set release version in .deb control files
```

Comment:
This line will be hard to maintain I guess.
Currently we're only writing about `.deb` files, but what about a later mac os x installer or other formats.
I guess they will require a modification of their own control files.
Therefore I suggest to write this more generic.

Reply:
I don't want to be more generic. The purpose of this page is that you have a check-list and don't forget anything. Therefore I'm willing to update it with any additional control file required. I don't think it will be that many. maybe flatpak, snap, rpm, OSX installer but that should be about it.